systemtest/build.xml

```diff
@@ -40,6 +40,24 @@
 			<arg value="https://github.com/AdoptOpenJDK/openjdk-systemtest.git" />
 			<arg value="${SYSTEMTEST_ROOT}/openjdk-systemtest" />
 		</exec>
+		<if>
+			<equals arg1="${TARGET}" arg2="openj9" />

```

Comment:
What is the TARGET value? I'm confused with this variable.

Reply:
The TARGET value is either 'opnej9' or 'system' for sytstemtest. It is what is passed in to maketest.sh in JenkinsfileBase.